src/app/js/admin/fields/FieldForm.js

```diff
@@ -14,8 +14,7 @@ import {
     saveField,
 } from './';
 
-import { fromFields } from '../../selectors';
-
+import { fromFields } from '../';
```

Comment:
Yes, and index is the index of the directory, it should be used to import the module represented by the directory, it is weird for a module to import itself.